### DISCUSSION TITLE
Is there a place for community builds?

### DISCUSSION THREAD

**Discussion by michaldobrodenka** [dotnet/runtime#92267] [General]
I build (from time to time) dotnet for armv6 and put it in docker image, which I use to publish self contained apps for armv6 cpu - https://hub.docker.com/r/taphome/dotnet7armv6

Is there a page with download links to unsupported builds? (eg armv6 or s390x)

Maybe someone do it regularly

**Comment by MichalPetryka** [comment]
> Is there a page with download links to unsupported builds? (eg armv6 or s390x)

I guess those would be the Linux distro repositories? IIRC the Fedora ones had such builds.
